clipper_admin/check_and_write_deps.py

```diff
@@ -28,11 +28,11 @@ def _write_out_dependencies(directory, conda_dep_fname, pip_dep_fname,
 
     with open(conda_dep_abs_path, 'wb') as f:
```

Comment:
Revert your changes and change the `open()` call to not open the file as a binary file:

```py
with open(conda_dep_abs_path, 'w') as f:
```